1-exercises/F-array-forEach/exercise.js

```diff
@@ -8,7 +8,15 @@
 */
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
-
+let newArray = arr.forEach((number) => {
+  if (number % 3 === 0) {
+    console.log("Fizz");
+  } else if (number % 5 === 0) {
+    console.log("Buzz");
+  } else {
```

Comment:
Please read the instruction again, Andrius. There are 3 conditions. You have successfully done first and second condetions but could add the last condition as well and print "FizzBuzz" when needed. Do you still think you need your last else???
Also, it's a good practice if in the future you could use ternary operator instead of if else chaining.